docs/apiref/dataset_iterators.rst

```diff
@@ -15,7 +15,8 @@ Concrete DatasetIterator classes.
 
 .. autoclass:: deeppavlov.dataset_iterators.kvret_dialog_iterator.KvretDialogDatasetIterator
 
-.. autoclass:: deeppavlov.dataset_iterators.morphotagger_iterator.MorphoTaggerDatasetIterator
+.. automodule:: deeppavlov.dataset_iterators.morphotagger_iterator
```

Comment:
I would prefer to the previous option in order to fine control of docstrings rendering.